modMcf/test/org/aion/mcf/account/AccountManagerTest.java

```diff
@@ -0,0 +1,111 @@
+/*
+ ******************************************************************************
+ * Copyright (c) 2017-2018 Aion foundation.
```

Comment:
I think this should use this header instead:

https://github.com/aionnetwork/aion/wiki/Aion-License-Header#2-new-functionality-developed-by-the-aion-foundation-will-have-the-license-below